lib/stripe/stripe_object.rb

```diff
@@ -32,7 +32,7 @@ def self.construct_from(values, opts={})
     # considered to be equal if they have the same set of values and each one
     # of those values is the same.
     def ==(other)
-      @values == other.instance_variable_get(:@values)
+      other.is_a?(StripeObject) && @values == other.instance_variable_get(:@values)
```

Comment:
If the other object is not a `StripeObject`, then it may not have a `@values` instance variable initialized.